src/lang/en-US.ts

```diff
@@ -968,6 +971,21 @@ const DataPlaneAuthReq: ResolvedIntlConfig['messages'] = {
     'dataPlaneAuthReq.waiting.message': `Please wait while we authorize access to {catalogPrefix}. You will be redirected shortly.`,
 };
 
+const SchemaEditor_Collection: ResolvedIntlConfig['messages'] = {
+    'schemaEditor.fields.label': `Schema`,
+    'schemaEditor.key.label': `Key`,
+    'schemaEditor.key.helper': `Ordered JSON-Pointers which define how a composite key may be extracted from a collection document.`,
+    'schemaEditor.table.empty.header': `No fields to display.`,
+    'schemaEditor.table.empty.message': `We were unable to generate a table from the current schema. Please update the schema.`,
+    'schemaEditor.error.title': `Schema Invalid`,
+    'keyAutoComplete.keys.invalid.message': `Field is not a valid key. Please remove or update the schema.`,
+    'keyAutoComplete.keys.invalid.message.readOnly': `Field is not a valid key. Please update the schema.`,
+    'keyAutoComplete.keys.missing.title': `Key is empty`,
+    'keyAutoComplete.keys.missing.message': `Please provide a key before continuing or you may face issues.`,
+    'keyAutoComplete.noOptions.message': `Without a valid schema we cannot provide options for the key. Please fix schema.`,
+    'keyAutoComplete.noUsableKeys.message': `No fields in the schema are valid keys. Please update schema.`,
+};
```

Comment:
@oliviamiannone - This new section is all the content related to the new editor. There are screenshots in the PR description.

Reply:
A few thoughts:
* Since we don't (probably CAN'T, realistically) go into detail of what's a valid key, is there a link to docs anywhere here (namely, https://docs.estuary.dev/concepts/collections/#schema-restrictions)? I think that'd be helpful for some of these messages
* Grammar fixes (hyphen and that vs which): should be "Ordered JSON Pointers that"
* not sure if this aligns with behavior, but for `keyAutoComplete.keys.missing.message`, might be better to be more blunt right "All collections require a key. Please provide a key to continue."

Reply:
Yup - I was thinking the same thing so there is not a link to the specification of the collection. I tried working in a specific link for the docs on keys but it was pretty tough to make it work with the current design.

![image](https://user-images.githubusercontent.com/270078/236237853-dd7957a2-c7ce-4b9f-ab65-a27d803af4c9.png)

Reply:
Resolved the last point and updated message. Makes more sense to just be blunt here.